apm-sniffer/apm-sdk-plugin/jdbc-commons/src/main/java/org/apache/skywalking/apm/plugin/jdbc/define/StatementEnhanceInfos.java

```diff
@@ -73,6 +73,22 @@ public void setParameter(int index, final Object parameter) {
         parameters[index] = parameter;
     }
 
+    public String getFullSql() {
+        StringBuilder resultSql = new StringBuilder();
+        int index = 0;
+        int startPos = 0;
+        int findPos = 0;
+        while ((findPos = sql.indexOf("?", startPos)) > 0) {
+            resultSql.append(sql.substring(startPos, findPos));
+            resultSql.append("'");
+            resultSql.append(parameters[index++]);
```

Comment:
It's unsafe to parse the SQL using `?`, are there possibilities when the sql includes `?` itself in the value, not a parameter placeholder?

`select * from news_table where author=? and title like '%?'`, I know it's a little tricky, just to remind the risk here

Reply:
My point is not to try to parse the SQL, store the parameters in another tag

Reply:
Parsing SQL is the next generation after ShardingShpere ( @tristaZero and  @terrymanu ) provides the real SQL Parser.  We don't do this by ourselves.

Reply:
@wu-sheng @kezhenxu94 ACK，i will store the parameters in another tag.